Swift Vocabulary/Swift Vocabulary/DefinitionViewController.swift

```diff
@@ -16,7 +16,9 @@ class DefinitionViewController: UIViewController {
         // Do any additional setup after loading the view.
     }
     
-
+    @IBOutlet var definitionTextView: UITextView!
+    
+    @IBOutlet var wordLabel: UILabel!
```

Comment:
`IBOutlets` should be placed before the `viewDidLoad()` and you don't need the extra line space between them